pkgs/tools/misc/hex/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, testers
+, hex
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hex";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "sitkevij";
+    repo = "hex";
+    rev = "v${version}";
+    sha256 = "sha256-mxKjiciejnOTbSkCzOWdAtysRAnEv4JgntPS1qM9og8=";
```

Since [this RFC](https://github.com/NixOS/rfcs/pull/131) is soon to be implemented, we probably should go with `hash` here.
```suggestion
    hash = "sha256-mxKjiciejnOTbSkCzOWdAtysRAnEv4JgntPS1qM9og8=";
```

---

Sounds good :+1:

---

> Since https://github.com/NixOS/rfcs/pull/131 is soon to be implemented

This isn't true, it doesn't even have a shepherd yet. Why do you think this is the case, is there a misunderstanding somewhere?

---

I think they just mean SRI style hashes are something that we're moving towards.